CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+## 0.2 (02 November 2016)
+
+* Added Initial Schedules Support (#48)
+* Added Initial Create Group endpoint (#69)
+* Added Connection Credentials for publishing datasources (#80)
+* Added Pager object for handling pagination results and sample (#72, #90)
+* Added ServerInfo endpoint (#84)
+* Code Cleanup
```

Comment:
It's a good thing that we're at the point of development where we don't list every little cleanup, right? 🎉